measures/emissions_reporting/measure.rb

```diff
@@ -504,81 +534,191 @@ def run(runner, user_arguments)
     end
 
     # get run period natural gas values
-    annual_natural_gas_emissions_co2e_kg = 0
+    annual_natural_gas_emissions_co2e_kg = 0.0
+    annual_natural_gas_emissions_nox_kg = 0.0
+    annual_natural_gas_emissions_co_kg = 0.0
+    annual_natural_gas_emissions_pm_kg = 0.0
+    annual_natural_gas_emissions_so2_kg = 0.0
     natural_gas_query = "SELECT VariableValue FROM ReportMeterData WHERE ReportMeterDataDictionaryIndex IN (SELECT ReportMeterDataDictionaryIndex FROM ReportMeterDataDictionary WHERE VariableType='Sum' AND VariableName='#{gas}:Facility' AND ReportingFrequency='Run Period' AND VariableUnits='J') AND TimeIndex IN (SELECT TimeIndex FROM Time WHERE EnvironmentPeriodIndex='#{env_period_ix}')"
     puts natural_gas_query
     natural_gas_values = sql_file.execAndReturnVectorOfDouble(natural_gas_query).get
     if natural_gas_values.empty?
-      runner.registerWarning('Unable to get hourly timeseries facility natural gas use from the model, the model may not use gas.  Cannot calculate emissions.')
+      runner.registerWarning('Unable to get hourly timeseries facility natural gas use from the model, the model may not use gas. Cannot calculate emissions.')
     else
-      annual_natural_gas_emissions_co2e_kg = natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_co2e_kg_per_kbtu }.sum
+      annual_natural_gas_emissions_co2e_kg += natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_co2e_kg_per_kbtu }.sum
+      annual_natural_gas_emissions_nox_kg += natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_nox_kg_per_kbtu }.sum
+      annual_natural_gas_emissions_co_kg += natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_co_kg_per_kbtu }.sum
+      annual_natural_gas_emissions_pm_kg += natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_pm_kg_per_kbtu }.sum
+      annual_natural_gas_emissions_so2_kg += natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_so2_kg_per_kbtu }.sum
     end
     runner.registerInfo("Annual hourly natural gas emissions (kg CO2e): #{annual_natural_gas_emissions_co2e_kg.round(2)}")
     runner.registerValue('annual_natural_gas_ghg_emissions_kg', annual_natural_gas_emissions_co2e_kg)
+    runner.registerInfo("Annual hourly natural gas NOx emissions (kg): #{annual_natural_gas_emissions_nox_kg}")
```

Comment:
What makes this "hourly"? Does it just mean that you manually summed the hourly timeseries values vs. getting the E+ annual value?